engine/class_modules/monk/sc_monk.cpp

```diff
@@ -171,6 +173,15 @@ void monk_action_t<Base>::apply_buff_effects()
   parse_effects( p()->buff.press_the_advantage );
 
   // Conduit of the Celestials
+  parse_effects( p()->buff.flight_of_the_red_crane );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr );
```

Comment:
The spell data pointed to by `buff.heart_of_the_jade_serpent_cdr` carries all of the effects used by the Heart of the Jade Serpent effect. Effect toggling is scripted via the spell pointed to by `buff.heart_of_the_jade_serpent_cdr_celestial`.

The CDR effect likely autoparses fine, but the channel time reduction may not.